content/guides/secure-payments-by-bookingsync.md

```diff
@@ -0,0 +1,121 @@
+# Secure Payments By BookingSync
+
+1. TOC
+{:toc}
+
+## Preface
+
+If you have an integration with BookingSync and you create bookings on your side according to the instant booking flow (i.e. a booking gets confirmed only when the credit card payment is executed successfully) and you don't have PCI DSS compliant platform for collecting credit card payments, you can use our Secure Payments By BookingSync flow for this purpose.
+
+## Requirements
+
+To handle integration with Secure Payments By BookingSync you need to have an application with `payments_write` or `payments_write_owned` scope. We recommend using `payments_write_owned` for this purpose to make sure the payment is not editable by anyone else, only by your application.
+
+Also, the rental for which the booking will be created requires to be instantly bookable and the it needs to have a payment gateway assigned (which will be used for executing the credit card payment) which supports the currency that is set for the rental.
+
+## Instant Booking Flow
+
+The primary idea behind Instant Booking Flow is to require successful credit card payment to confirm the booking.
+
+Here are the steps that we recommend to follow for this flow:
+
+* Create a tentative booking
+* Create a tentative instant booking payment
+* Send credit card data and payment id to our PCI DSS platform for executing payment, which will confirm both booking and payment if the payment is successfully executed
+
+Let's focus on each point separately.
+
+## Creating A Tentative Booking
+
+Instant Booking flow requires payment to confirm the booking, so we recommend creating a tentative booking and leaving the confirmation to our PCI DSS platform. The confirmation process should take up to few minutes, so we recommend to set tentative expire date to 5 or 10 minutes from now. To create a tentative booking you need to provide 2 params (besides the requires one):
```

Comment:
what about? `confirmation to our PCI DSS platform` > `confirmation to our PCI DSS payment platform`